### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by svitalis123**
### I accomplished

- [ ] Create a repository, create a dev branch, and make it default
- [ ] set up the project by installing react app and linters
- [ ] Get API and filter more details
- [ ] Test the pure functions

Here is my loom [video](https://www.loom.com/share/9a2411e8409d42c89e45e458f1a4ce84)

### CODE REVIEW

File: src/Home.js [hunk 1]
```diff
@@ -1,9 +1,54 @@
...
+                key={item.id}
+                datas={item}
+              />
+            ))
+          }
+        </div>
+      </div>
     </div>
   );
 }
```
Comment (Mihndim2020) [inline src/Home.js:47]: - [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md)for this project,  " ` Home page: When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.` ", kindly add a component, e.g a search box that can be used to filter what is displayed on the home page(`Home page has a list of categories that could be filtered using given parameters, e.g. category name.`). For example, you should be able to filter components based on their names.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @svitalis123, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark:  Good job with tests coverage
:heavy_check_mark: Clean and Organized code

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Home.js (hunk 1 above)
Comment (svitalis123) [reply src/Home.js:47]: ## Thank you
# i have resolved it by including a search box in the home to filter by continent name

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @svitalis123 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great Job implementing the changes provided by the previous reviewer ✅
- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**